composer.json

```diff
@@ -12,14 +12,19 @@
     "license": "AGPL-3.0-or-later",
     "authors": [
         {
-            "name": "Oliver Klee",
-            "email": "oliver@phplist.com",
-            "role": "Developer"
+            "name": "Xheni Myrtaj",
+            "email": "xheni@phplist.com",
+            "role": "Maintainer"
         },
         {
             "name": "Sam Tuke",
             "email": "mail@samtuke.com",
             "role": "Developer"
```

Comment:
Could you also please change Sam to "Former developer" as well? He's left the project, too.